### ISSUE TITLE
Are you accepting donations as a form of support for the product?

### ISSUE THREAD

**Issue by demahum** [lensapp/lens#2502]
I would like to contribute and show appreciation and love for the product, but don't want to do anything that is already listed for [contributing options](https://docs.k8slens.dev/latest/contributing/).

Is there an option to donate some money to the product? Something like https://www.buymeacoffee.com/ would be cool to have.

**Comment by andrewnazarov** [comment]
Or Github sponsorship might be an option:)

**Comment by demahum** [comment]
> Or Github sponsorship might be an option:)

Agreed. Although it would be better to find something more inclusive. [GitHub sponsorship](https://github.com/sponsors) is available in only 36 countries at the moment, mine is not on the list.

**Comment by dzmitry-lahoda** [comment]
where is a button to donate? i want you to take some money for several coffees:).  preferably in crypto like solana so, but can do fiat too.
